src/lean_spec/subspecs/networking/messages.py

```diff
@@ -52,11 +52,11 @@ class Status(StrictBaseModel):
 """
 
 BlocksByRootResponse = Annotated[
-    list[SignedBlock],
+    list[SignedBlockAndVote],
```

Comment:
this change is not needed, Vote is packed with the Block only because of signature constrains how ever its processed and packed independently by a future block (hopefully the next)

ahh ok but we will the full message to correctly verify signature, so nevermind